packages/bruno-app/src/providers/ReduxStore/slices/collections/actions.js

```diff
@@ -117,22 +117,23 @@ export const sendRequest = (item, collectionUid) => (dispatch, getState) => {
       })
       .then(resolve)
       .catch((err) => {
+        const errorMessage = err.message ?? 'Something went wrong';
+
+        const errorResponse = {
+          status: 'Error',
+          isError: true,
+          error: errorMessage,
+          size: '0',
+          duration: '0'
+        };
+
         dispatch(
           responseReceived({
             itemUid: item.uid,
             collectionUid: collectionUid,
-            response: null
+            response: errorResponse
           })
         );
-
-        if (err && err.message === "Error invoking remote method 'send-http-request': Error: Request cancelled") {
-          console.log('>> request cancelled');
-          return;
-        }
-
```

Comment:
We need to keep above lines and should not show any error if the request was manually cancelled using the cancel button.